tests/tests_core.py

```diff
@@ -9,8 +9,8 @@
 class TestsCore(unittest.TestCase):
 
     # __init__
-    def test_init_givenNoEnvironmentVariable_coreIsNotInitialized(self):
```

Comment:
@evhen14 I think we should not remove this test (see the associated comment above). Instead add another test for IOError (like you already have)